packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1000,6 +1055,7 @@ class CupertinoTimerPicker extends StatefulWidget {
   final CupertinoTimerPickerMode mode;
 
   /// The initial duration of the countdown timer.
+  @Deprecated('Migration to use controller')
```

Comment:
Be more specific here. This string shows up when you highlight the lint in IDEs etc.